src/pages/MyPage/MyPage.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from 'react';
+import BoxData from './BoxData';
+import './MyPage.scss';
+
+const MyPage = () => {
+  const CARD_LIST = [
+    {
+      title: 'ORDER',
+      content: '주문내역',
+    },
+    {
+      title: 'ORDER',
+      content: '주문내역',
+    },
+    {
+      title: 'ORDER',
+      content: '주문내역',
+    },
+    {
+      title: 'ORDER',
+      content: '주문내역',
+    },
+    {
+      title: 'ORDER',
+      content: '주문내역',
+    },
+    {
+      title: 'ORDER',
+      content: '주문내역',
+    },
+  ];
+
+  return (
+    <div className="Mypage">
+      <div className="container">
+        <div>
+          <ul className="textGroup">
+            <li>
+              <span>가용적립금 </span>
+            </li>
+            <li>
+              <span className="savedMoney">0원 조회</span>
+            </li>
+          </ul>
+        </div>
+        <ul className="cardList">
+          {CARD_LIST.map((cardList, i) => {
+            return <BoxData key={i} {...cardList} />;
+          })}
```

Comment:
```suggestion
          {CARD_LIST.map((card, i) => {
            return <BoxData key={i} {...card} />;
          })}
```

상수데이터 활용, 반복되는 UI Array.map으로 처리, spread 문법 활용 다 너무 잘하셨습니다 👍

프리코스, 파운데이션때보다 많이 실력이 늘고 계신 것 같네요 좋습니다💪

다만, CARD_LIST라는 배열의 각자 요소들의 이름이 또다시 cardList인것보다는 card라고 명칭을 하는게 좀 더 알맞은 표현같네요! 이름에 의미가 잘 담기도록 계속 고민해주시고 지금처럼 계속 학습 잘 해나가시면 될 것 같습니다!!